bin/base.sh

```diff
@@ -1,11 +1,25 @@
 set -e
 
 # shellcheck disable=SC2034
-ch_bin="$(cd "$(dirname "$0")" && pwd)"
+ch_bin=$(cd "$(dirname "$0")" && pwd)
 
-libexec="$(cd "$(dirname "$0")" && pwd)"
+libexec=$(cd "$(dirname "$0")" && pwd)
 . "${libexec}/version.sh"
 
+progname=$(basename "$0")
+
+debug () {
+    echo "$1" 1>&2
```

Comment:
`debug` should be silent unless we're in verbose mode. Previously it tested `$verbose` but I'm open to argument on how to make it happen.

Reply:
The only script that uses (and sets) the `--verbose` argument is `ch-fromhost`. So the reference to `$verbose` in the `base.sh` is undefined.